src/tools/perf/api/libperf.h

```diff
@@ -84,7 +84,8 @@ enum ucx_perf_test_flags {
     UCX_PERF_TEST_FLAG_TAG_UNEXP_PROBE  = UCS_BIT(5), /* For tag tests, use probe to get unexpected receive */
     UCX_PERF_TEST_FLAG_VERBOSE          = UCS_BIT(7), /* Print error messages */
     UCX_PERF_TEST_FLAG_STREAM_RECV_DATA = UCS_BIT(8), /* For stream tests, use recv data API */
-    UCX_PERF_TEST_FLAG_FLUSH_EP         = UCS_BIT(9)  /* Issue flush on endpoint instead of worker */
+    UCX_PERF_TEST_FLAG_FLUSH_EP         = UCS_BIT(9), /* Issue flush on endpoint instead of worker */
+    UCX_PERF_TEST_FLAG_INTERRUPT        = UCS_BIT(10) /* Request interrupt notification support. */
```

Comment:
INTERRUPT -> WAKEUP in all places
to align with UCP terminology

Reply:
Done, the issue is both `-w` `-W` are used, selected `-I` to be mnemonic with interrupt.